constraint_utils/constraint_basics.py

```diff
@@ -685,14 +708,35 @@ def annotate_constraint_groupings(ht: Union[hl.Table, hl.MatrixTable],
 # Model building
 def build_coverage_model(coverage_ht: hl.Table) -> (float, float):
     """
-    Calibrates coverage model (returns intercept and slope)
+    Calibrate coverage model (returns intercept and slope).
```

Comment:
`Calibrate coverage model` isn't very descriptive, what is it modeling? I also don't think you need (returns intercept and slope) it's specified by the return below